cli/unifai_cli/api/base.py

```
@@ -22,71 +25,59 @@ def __init__(self, base_url: str):
     def _url(self, parent: str, route: str) -> str:
         return f"{self.base_url}{self.api_prefix}/{parent}/{route}"
 
-    def _auth_headers(self, user_id: Optional[str] = None) -> dict[str, str]:
-        uid = self._effective_user_id(user_id)
-        if not uid:
-            return {}
-        return {AUTH_USER_HEADER: uid}
-
-    def _get(self, parent: str, route: str, params: dict = None,
-             user_id: Optional[str] = None) -> Any:
+    def _get(self, parent: str, route: str, params: dict = None) -> Any:
```

Ruff (0.15.17)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)